wrangler-api/src/main/java/io/cdap/wrangler/api/lineage/Mutation.java

```diff
@@ -16,60 +16,242 @@
    */
-  public String column() {
-    return column;
+  public String readable() {
+    return readable;
   }
 
   /**
-   * @return the type of mutation being applied on the <code>column</code>.
+   * @return a {@link List} of {@link Relation} associated with the transformation.
    */
-  public MutationType type() {
-    return type;
+  public List<Relation> relations() {
+    return relations;
   }
 
   /**
-   * @return the optional description for the mutation.
+   * @return a instance of {@link Mutation.Builder}
    */
-  public String description() {
-    return description;
+  public static Mutation.Builder builder() {
+    return new Mutation.Builder();
   }
 
   /**
-   * Generates a <code>JsonElement</code> object representation of <code>Mutation</code>
-   * @return the object representation of <code>Mutation</code> as <code>JsonElement</code>
+   * Builder to create Mutation.
    */
-  public JsonElement toJson() {
-    JsonObject object = new JsonObject();
-    object.addProperty("column", column);
-    object.addProperty("type", type.name());
-    object.addProperty("description", description);
-    return object;
+  public static class Builder {
+    private final List<Relation> relations;
+    private String description;
+
+    /**
+     * A builder constructor.
+     */
+    public Builder() {
+      this.relations = new ArrayList<>();
+    }
+
+    /**
+     * An easy way to created a formatted string of transformation.
+     *
+     * @param format a format string.
+     * @param args Arguments referenced by the format specifiers in the format string.
+     * @return a instance of {@link Mutation.Builder}.
+     */
+    public Mutation.Builder readable(String format, Object ... args) {
+      this.description = String.format(format, args);
+      return this;
+    }
+
+    /**
+     * A variation to specify the readable string for transformation.
+     *
+     * @param format a format string.
+     * @param args Arguments referenced by the format specifiers in the format string.
+     * @return a instance of {@link Mutation.Builder}.
+     */
+    public Mutation.Builder readable(String format, List<Object> args) {
+      readable(format, args.toArray());
+      return this;
+    }
+
+    /**
+     * Specifies a relation that need to be dropped.
+     *
+     * @param sources a list of sources to be dropped from lineage.
+     * @return a instance of {@link Mutation.Builder}.
+     */
+    public Mutation.Builder drop(Many sources) {
+      relations.add(new Relation(uuid(), sources.columns(),
+                                 Collections.emptyList(), Relation.Type.DROP));
+      return this;
+    }
+
+    /**
+     * Specifies a relation that has no cause but effect.
+     *
+     * @param targets a list of targets.
+     * @return a instance of {@link Mutation.Builder}.
+     */
+    public Mutation.Builder generate(Many targets) {
```

Comment:
Is there any specific reason to use method name `generate`(equivalent of CREATE) and `relation` (equivalent of STANDARD). Would it be a good idea to use create and standard method names?

Reply:
I think `relation` is fine for STANDARD, since most of the directives will provide clear relation, will change `generate` to `create`